simulation/halsim_ds_socket/src/main/native/cpp/main.cpp

```diff
@@ -97,6 +103,7 @@ static void SetupTcp(wpi::uv::Loop& loop) {
     gDSConnected = true;
 
     client->data.connect([t](Buffer& buf, size_t len) {
+      gDSLastPacketTime = HAL_GetFPGATime(nullptr);
```

Comment:
With the simplification of the callback, the gDSLastPacketTime can be removed along with the HAL include.